test/e2e/common_test.go

```diff
@@ -377,7 +408,9 @@ func GetPortLock(port string) storage.Locker {
 // collisions during parallel tests
 func GetRandomIPAddress() string {
 	// To avoid IP collisions of initialize random seed for random IP addresses
-	rand.Seed(time.Now().UnixNano())
+	seed := time.Now().UnixNano()
+	logrus.Warnf("rand.Seed @ test/e2e/common_test.go:GetRandomIPAddress: %#v", seed)
```

Comment:
IMHO logging is unnecessary here.

Reply:
_all_ of these `rand.Seed` logs are clearly unsuitable for production, they were just attempts to figure out which of the calls ultimately matters (but see #12155, it seems not to work).

Reply:
Yep, I suspected as much, just wanted to make sure they're not forgotten about.